.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+  Thanks for submitting a pull request!
+  We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request. The three fields below are mandatory.
+
+  Before submitting a pull request, please make sure the following is done:
+
+  1. Fork [the repository](https://github.com/solidjs/solid) and create your branch from `main`.
```

Comment:
Generally we don't use yarn, but npm.. there is no yarn.lock or anything.
You should run npm run build before running any tests as it copies some files in that are required for Solid to work. Otherwise this looks pretty reasonable.

Reply:
@ryansolid  Thanks Ryan, Solved :100: